boot/freeldr/bootsect/ext.S

```diff
@@ -0,0 +1,490 @@
+/*
+ * PROJECT:     FreeLoader
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     EXTFS volume boot sector
+ * COPYRIGHT:   Copyright 2002-2003 Brian Palmer <brianp@sginet.com>
+ *              Copyright 2024 Daniel Victor <ilauncherdeveloper@gmail.com>
+ */
+
+#include <asm.inc>
+#include <freeldr/include/arch/pc/x86common.h>
+
+#define BP_REL(x) ss:[bp+x-offset start]
+
+#define EXT_INODE_BLOCKS                12
+#define EXT_INODE_INDIRECT_BLOCKS       3
+#define EXT_ROOT_INO			        2
+#define EXT_BLOCK_SIZE	                1024
+#define EXT_INODE_ADDRESS		        HEX(9000)
+#define EXT_DIRECTORY_ADDRESS	        HEX(A000)
+#define EXT_BLOCK_ADDRESS	            HEX(B000)
+#define EXT_BLOCK2_ADDRESS	            HEX(C000)
+#define EXT_INODE_TYPE_MASK             HEX(F000)
+#define EXT_INODE_TYPE_REGULAR          HEX(8000)
```

Comment:
Please indent the values in a consistent manner.
And same remark for the code comments below as well.